docs/platform-external/installing.md

```diff
@@ -0,0 +1,433 @@
+
+- the address `api-int.clusterDomain` must point to the internal Load Balancer address.
+- the Load Balancers must support hairpin connections
+
+## Preparing the installation
+
+The `External` platform type configuration is created in this section. The steps below describe how to create the `install-config.yaml` with the required fields, and the steps used to customize the providers' manifests before generating the deployment/ignition configuration/ignition.
+
+### Create the install-config.yaml
+
+Follow the steps to [Manually create the installation configuration file](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-initializing-manual_installing-platform-agnostic), customizing the `platform` object, setting the type to `external`, and the `platformName` to the cloud provider's name:
+
+> Q: Installer PR is in progress. Will the survey collect the `platformName` field?
+
+> Q: Do we need to add ./openshift-install create install-config?
+
+```yaml
+platform:
+  external:
+    platformName: "providerName"
+```
+
+### Create manifests
+
+Once the `install-config.yaml` was created, you can generate the manifests for the deployment by running the command:
+
+```bash
+./openshift-install create manifests
+```
+
+The `openshift/` and `manifests/` directory will be created.
+
+The steps below describe how to customize the OpenShift installation.
+
+The manifests directory will be copied to the bootstrap and applied when the core components are initialing the Control Plane.
+
+#### Create custom manifests for CCM
+
+In that stage, the Cloud Controller Manager (CCM) manifests must be created, as it is responsible to initialize the nodes. A set of tolerations must be set to the pod spec to ensure the controllers are initialized even if the node is not yet ready.
+
+```yaml
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: "NoSchedule"
+        - key: node.cloudprovider.kubernetes.io/uninitialized
+          operator: Exists
+          effect: NoSchedule
+        - key: node.kubernetes.io/not-ready
+          operator: Exists
+          effect: NoSchedule
+```
+
+The manifest below can be used as a Deployment configuration:
+
+```yaml
+# This a template that can be used as a base to run your own cluster controller manager in openshift, all you need is to:
+# - replace values between {{ and }} with your own ones
+# - specify a command to startup the CCM in your container
+# - define and mount extra volumes if needed
+# This example defines the CCM as a Deployment, but a DaemonSet is also possible as long as Pod's template is defined in the same way.
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: {{ cloud provider name }}-cloud-controller-manager # replace me
+  namespace: provider-cloud-controller-manager # replace me
```

Comment:
```suggestion
  namespace: {{ cloud provider name }}-cloud-controller-manager # replace me
```

When I build this template, I took the assumption we will re-use the operator's namespace. But if our recommendation is to create a new namespace, I think we should provide a template for it, for the service account, and the RBAC (I will make a suggestion)